litr/build.gradle

```diff
@@ -1,16 +1,16 @@
 apply plugin: 'com.android.library'
 
-apply from:'../constants.gradle'
+apply from: "$rootDir/gradle/publishing.gradle"
+
+description 'LiTr media transformation library for Android'
 
 android {
-    compileSdkVersion project.ext.compileSdkVersion
-    buildToolsVersion project.ext.buildToolsVersion
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
```

Comment:
Can we remove some entries from `constants.gradle`?

Reply:
This is a good catch, I had meant to delete `constants.gradle` entirely. 😄